src/jobs/models/jobManager.js

```diff
@@ -34,16 +38,35 @@ module.exports.reloadCronJobs = async () => {
     const configData = await configHandler.getConfig();
     try {
         const jobs = await databaseConnector.getJobs(contextId);
-        jobs.forEach(async function (job) {
+        for (const job of jobs) {
```

Comment:
Unless you have a reason, I wouldn't change that. (I know you changed bc of await, but no need for await).

Reply:
reverted removed the async